src/domain/discounts/details/use-promotion-settings.tsx

```diff
@@ -0,0 +1,333 @@
+import { getErrorMessage } from "../../../utils/error-messages"
+
+type displaySetting = {
+  title: string
+  description: ReactNode
+  actions: ActionType[]
+}
+
+const DisplaySettingsDateDescription = ({ date }: { date: Date }) => (
+  <div className="flex text-grey-50 inter-small-regular ">
+    {moment(date).format("ddd, DD MMM YYYY")}
+    <span className="flex items-center ml-3">
+      <ClockIcon size={16} />
+      <span className="ml-2.5">{moment(date).format("UTC HH:mm")}</span>
+    </span>
+  </div>
+)
+
+const CommonDescription = ({ text }) => (
+  <span className="text-grey-50 inter-small-regular">{text}</span>
+)
+
+const getConditionSettings = (condition: any, openWithItems) => {
+  switch (condition.type) {
+    case "products":
+      return {
+        title: "Products",
+        description: "test",
+        actions: [
+          {
+            label: "Edit",
+            icon: <EditIcon size={20} />,
+            onClick: () => openWithItems(["conditions"]),
+          },
+        ],
+      }
+    case "product_types":
+      return {
+        title: "Type",
+        description: "test",
+        actions: [
+          {
+            label: "Edit",
+            icon: <EditIcon size={20} />,
+            onClick: () => openWithItems(["conditions"]),
+          },
+        ],
+      }
+    case "product_collections":
+      return {
+        title: "Collection",
+        description: "test",
+        actions: [
+          {
+            label: "Edit",
+            icon: <EditIcon size={20} />,
+            onClick: () => openWithItems(["conditions"]),
+          },
+        ],
+      }
+    case "product_tags":
+      return {
+        title: "Tag",
+        description: "test",
+        actions: [
+          {
+            label: "Edit",
+            icon: <EditIcon size={20} />,
+            onClick: () => openWithItems(["conditions"]),
+          },
+        ],
+      }
+    case "customer_groups":
+      return {
+        title: "Customer group",
+        description: "test",
+        actions: [
+          {
+            label: "Edit",
+            icon: <EditIcon size={20} />,
+            onClick: () => openWithItems(["conditions"]),
+          },
+        ],
+      }
+  }
+  return null
+}
+
+const usePromotionSettings = (promotion: Discount, openWithItems) => {
+  const updateDiscount = useAdminUpdateDiscount(promotion.id)
+
+  const notification = useNotification()
+
+  const conditions: displaySetting[] = []
+
+  conditions.push({
+    title: "Regions",
+    description: (
+      <CommonDescription
+        text={promotion.regions.map((r) => r.name).join(", ")}
+      />
+    ),
+    actions: [
+      {
+        label: "Edit",
+        icon: <EditIcon size={20} />,
+        onClick: () => openWithItems(["general"]),
+      },
+      {
+        label: "Delete setting",
+        icon: <TrashIcon size={20} />,
+        variant: "danger",
+        disabled: true,
+        onClick: () => {},
+      },
```

Comment:
nit(non-blocking): Would it make sense to omit "Delete setting" if it's unavailable instead of showing it as disabled.

Reply:
Maybe, I have removed them for now, you can see how you like it. I initially had them because the actionables seemed a bit empty without. However, I think if we add them we should also add a tooltip that says you cannot delete this setting. I think then it would make sense, otherwise we shouldn't have them. 

wdyt?